src/02_spec/07_modules/dem_uart/index.rst

```diff
@@ -0,0 +1,29 @@
+UART Device Emulation Module (DEM_UART)
+=======================================
+
+.. figure:: img/osd_dem_uart.*
+   :alt: Device Emulation Module UART
+
+The UART Device Emulation Module (DEM_UART) connects to the bus of a given CPU on one side,
+and to the Debug Interconnect on the other.
+Towards the CPU it behaves like a UART-16550 device, but instead of transmitting information
+over a UART-Interface it instead passes it to a host PC via the Debug Interconnect Interface.
+
+The DEM_UART_16550 component is responsible for emulating the behavior of a UART chip, while
+the DEM_UART component puts the received data into valid Debug Interconnect packets.
+The following UART registers are implemented and accessible via the bus, the address mapping
+and function are according to the `UART 16550D Datasheet <https://www.ti.com/lit/ds/symlink/pc16550d.pdf>`_
```

Comment:
We want to implement the 16550A component (not the D version) with the properties shown in the OpenRISC device tree file. Please add those to the documentation as well.